admin/src/App.jsx

```diff
@@ -0,0 +1,79 @@
+import { useState } from 'react'
+import DashboardModule from './components/DashboardModule'
+import Switch from './elements/Switch'
+import Button from './elements/Button'
+import Checkbox from './elements/Checkbox'
+import Tooltip from './elements/Tooltip'
+import SearchField from './elements/SearchField'
+import InputField from './elements/InputField'
+import FilterMenu from './elements/FilterMenu'
+import SortMenu from './elements/SortMenu'
+import RangeSlider from './elements/RangeSlider'
+import OptionButton from './elements/OptionButton'
+
+function App() {
+  const [count, setCount] = useState(0);
+
+  const testingFilterItems = {
+    0: 'Most reviews from external portals',
+    1: 'Best ratings from external portals',
+    2: 'Out best rating',
+    3: 'Recently updated'
+  };
+
+  // const createPost = fetch('http://liveagent.local/wp-json/wp/v2/posts', {
```

Comment:
remove such comments

Reply:
Not used module, only to demo UI elements – will be removed later